doc/landing-page/index.html

```diff
@@ -0,0 +1,259 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Prism - Bootstrap 4 Open Source Landing Page Template</title>
+    <meta name="description" content="Prism is a beautiful Bootstrap 4 template for open-source landing pages."/>
```

Comment:
title and description have to be updated.

Reply:
yes, updating